### PR TITLE
[WIP] Add support for batch handlers for accounts

### PR DESCRIPTION
**Pull request by sbellity**
* **Deprecation** Renamed `userHandlerOptions` to `options` in
notifyHandler

### CODE REVIEW

File: src/utils/notif-handler.js [hunk 1]
```diff
@@ -36,7 +37,7 @@ function getHandlerName(eventName) {
   return `${model}:${action}`;
 }
 
-function processHandlersFactory(handlers, userHandlerOptions = {}) {
+function processHandlersFactory(handlers, options = {}) {
```
Comment (michaloo) [inline src/utils/notif-handler.js]: I think for deprecation period we should support both names. Otherwise it's just breaking.

Comment (unity) [reply src/utils/notif-handler.js]: They are supported. Here it’s not a named argument so the internal name doesn’t matter no ?

Comment (michaloo) [reply src/utils/notif-handler.js]: sorry, I've commented on wrong line, you are absolutely right. The breaking change is in smartNotifierHandler, let me comment it.

File: src/utils/smart-notifier-handler.js [hunk 2]
```diff
@@ -90,17 +91,17 @@ function processHandlersFactory(handlers, userHandlerOptions) {
   };
 }
 
-
-module.exports = function smartNotifierHandler({ handlers = {}, userHandlerOptions = {} }) {
+module.exports = function smartNotifierHandler({ handlers = {}, options = {} }) {
```
Comment (michaloo) [inline src/utils/smart-notifier-handler.js]: This is the line which breaks from:
`{ handlers = {}, userHandlerOptions = {} }`
to
`{ handlers = {}, options = {} }`

**Pull request merged**